charts/iter8lib/templates/_metrics-common-istio.tpl

```diff
@@ -0,0 +1,116 @@
+{{- define "metrics.common.istio" }}        
```

Comment:
We do need tpl files. Perhaps, create a `testdata/sample-chart/templates/istio.metrics.tpl` file for testing purposes.